Zappy_IA_Report/DTC/Code/data.py

```
@@ -0,0 +1,62 @@
+import sys
+import pandas as pd
+from sklearn.tree import DecisionTreeClassifier
+import os
+import joblib
+import time
```

Pylint (3.3.7)

[convention] standard import "time" should be placed before third party imports "pandas", "sklearn.tree.DecisionTreeClassifier", "joblib"

(C0411)